analyzer/src/funTest/assets/projects/synthetic/spdx-packages-expected-output.yml

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
Commit message nit: Use a consistent plural, either "ymls" or "yml's", but not mixed.